tools/api-markdown-documenter/package.json

```diff
@@ -72,7 +72,7 @@
 		"@fluid-tools/markdown-magic": "file:../markdown-magic",
 		"@fluidframework/build-common": "^2.0.3",
 		"@fluidframework/build-tools": "^0.26.1",
-		"@fluidframework/eslint-config-fluid": "file:../../common/build/eslint-config-fluid",
+		"@fluidframework/eslint-config-fluid": "^4.0.0",
```

Comment:
https://dev.azure.com/fluidframework/public/_build/results?buildId=242582&view=logs&j=3dc8fd7e-4368-5a92-293e-d53cefc8c4b3&t=bfe27d5f-9219-57c0-0d67-1b2946a32921&l=73

Changed to `^4.0.0` for now.